src/components/Trade/PricesWebsocket.ts

```diff
@@ -0,0 +1,81 @@
+import axios from "axios";
+import websocket from "websocket";
+import { WebsocketPrice } from "../../types/trade";
+import { pairFromString } from "../../utils/general-utils";
+
+class PricesWebsocket {
+  public client: websocket.w3cwebsocket;
+
+  protected callback: (data: WebsocketPrice) => void = () => void 0;
+
+  constructor(initialPairs: string[]) {
+    const socket = new websocket.w3cwebsocket("wss://oracle.handle.fi/quotes");
```

Comment:
Please put the default `ORACLE_WS_QUOTES_ENDPOINT` in a config folder/file
```suggestion
  constructor(initialPairs: string[], overrides?: { websocketUrl?: string }) {
    const socket = new websocket.w3cwebsocket(overrides?.websocketUrl ?? ORACLE_WS_QUOTES_ENDPOINT);
```